r2i/onnxrt/engine.cc

```diff
@@ -60,6 +60,23 @@ RuntimeError Engine::SetModel (std::shared_ptr<r2i::IModel> in_model) {
   return error;
 }
 
+void Engine::CreateEnv() {
+  env = Ort::Env(this->logging_level, this->log_id.c_str());
+}
+
+void Engine::CreateSessionOptions() {
+  session_options = Ort::SessionOptions();
+  session_options.SetIntraOpNumThreads(this->intra_num_threads);
```

Comment:
Where are we setting the defaults. You should initialize every member of the class to a known state.